R/make_states.R

```diff
@@ -0,0 +1,39 @@
+make_state_map_one <- function(state, geometry = TRUE, epsg = alarm_epsg(state)) {
+
+    nd <- geomander::get_alarm(state = state, geometry = geometry, epsg = epsg)
+
+    nd <- nd %>%
+        summarise(dplyr::across(tidyselect::starts_with("pop"), sum),
```

Comment:
so one thing with R packages is that all thes dplyr calls have to have `dplyr::` in front, just like you've done for most of the rest of this function

Reply:
line 16 and 20 too